tests/test_app_basic.py

```
@@ -0,0 +1,164 @@
+        
+        response = client.get('/')
+        assert response.status_code == 200
+        # CSRFトークンがHTMLに含まれているか確認（実装依存）
+    
+    def test_app_imports(self):
+        """アプリケーションモジュールのインポートテスト"""
+        # リファクタリング後のモジュールがインポート可能か確認
+        try:
+            import services.session_service
+            import services.llm_service
+            import services.chat_service
```

Ruff (0.12.2)

`services.chat_service` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)